simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountListRegistryFragment.kt

```diff
@@ -43,6 +44,10 @@ class AccountListRegistryFragment : Fragment(R.layout.account_list_registry) {
   private val subscriptions = CompositeDisposable()
   private val listener: FragmentListenerType<AccountListRegistryEvent> by fragmentListeners()
 
+  private val requestLocationPermission = registerForActivityResult(
+    ActivityResultContracts.RequestPermission(),
+    ::getLocation
+  )
```

Comment:
Is this intended to be part of this change? It seems to have come in with the last merge, but I'm not sure how it's related.

Reply:
Yes, it is. I deleted that because it was being unused, but there are some commented code that is using that and that's why it was considered to be unused. However, I thought it would be better to restore it in case the code gets uncommented and in that way we can save some time.